docs/dev/tutorials/aa-daily-spend-limit.md

```diff
@@ -928,20 +1042,26 @@ After the error, we can rerun the code with a different ETH amount that doesn't
 If the transaction succeeds, the output would be like the following:
 
 ```shell
-l1TimeStamp:  1673530137
-resetTime:  1673529801
-limit:  5000000000000000
-available:  100000000000000
-New resetTime: 1673530575
+Account ETH limit is:  500000000000000
+Available today:  499780000000000
+L1 timestamp:  1683111266
+Limit will reset on timestamp:  1683111958
+Sending ETH transfer from smart contract account
+ETH transfer tx hash is 0x90f1ca06e6b407dfba75da2b0e9a7d06909c1c7d702f9da44fa5124ae5864dfc
+Transfer completed and limits updated!
+Account limit:  500000000000000
+Available today:  499670000000000
+Limit will reset on timestamp:: 1683111958
+Reset time was not updated as not enough time has passed
 ```
 
-The value `available` in the Limit struct was decremented, so now only 0.0001 ETH is available for transfer.
+The `available` value in the Limit struct was updated accordingly to the initial limit minus the amount we transfered.
 
-Since the `ONE_DAY` is set to 1 minute for this test, another transfer with any amount less than the limit is supposed to succeed accordingly after a minute instead of 24 hours. However, the second transfer would fail, and we would have to wait until the next L1 batch is sealed (around ten minutes on testnet) to make a successful transaction instead. To understand the reason behind this, we should know about a constraint of using `block.timestamp`.
+Since the `ONE_DAY` is set to 1 minute for this test in the `SpendLimit.sol` contract, you should expect it to reset after 60 seconds. However, we're using `block.timestamp` so the limit is only reset after a new L1 batch is sealed (around ten minutes on testnet). To understand the reason behind this, we should know about a constraint of using `block.timestamp`.
```

Comment:
```suggestion
Since `ONE_DAY` is set to 1 minute for this test in the `SpendLimit.sol` contract, you should expect it to reset after 60 seconds. However, we're using `block.timestamp` so the limit is only reset after a new L1 batch is sealed (around ten minutes on testnet). To understand the reason behind this, we should know about the constraints around using `block.timestamp`.
```